db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
```

Comment:
I think it's fine to remove this, since it's in the `gemfile.rb`, it's available throughout the project.